rclpy/rclpy/task.py

```diff
@@ -85,8 +85,12 @@ def result(self):
         """
         Get the result of a done task.
 
+        This raises an exception if one was set during the task.
```

Comment:
You can also use sphinx syntax:

```suggestion
        :raises: Exception if one was set during the task.
```